src/test/java/io/vertx/mqtt/test/client/MqttClientOutOfOrderAcksTest.java

```diff
@@ -93,18 +95,22 @@ private void clientSendThreePublishMessages(MqttQoS mqttQoS, TestContext context
   }
 
   @Before
-  public void before() {
+  public void before() throws InterruptedException {
+    vertx = Vertx.vertx();
```

Comment:
I see that the vertx instance is already created on the declaration line : 

`Vertx vertx = Vertx.vertx();`

Reply:
You are right